<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>community_window</class>
 <widget class="QWidget" name="community_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>社区</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>540</height>
    </rect>
   </property>
   <widget class="QPushButton" name="share_func">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>分享功能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="share_tool">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>分享工具</string>
    </property>
   </widget>
   <widget class="QPushButton" name="share_tunnel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>分享通道</string>
    </property>
   </widget>
   <widget class="QPushButton" name="share_command">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>分享命令</string>
    </property>
   </widget>
   <widget class="QPushButton" name="share_part">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>分享配件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>共享社区</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>其他分享功能暂未开放</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>其他分享功能暂未开放</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_addr">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>addr:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="button_refresh">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>7</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>状态：</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>561</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="button_list_widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>557</width>
       <height>427</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="button_change_server">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>7</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>更改</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
